cypress/integration/examples/aliasing.spec.js

```diff
@@ -0,0 +1,40 @@
+/// <reference types="cypress" />
+
+context('Aliasing', () => {
+  beforeEach(() => {
+    cy.visit('https://example.cypress.io/commands/aliasing')
```

Comment:
Update the URL to the Book Store app instead of 'https://example.cypress.io/commands/aliasing'